sdk/core/System.ClientModel/tests/client/Internal/ModelReaderWriterExtensions.cs

```diff
@@ -0,0 +1,264 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.Linq;
+using System.ClientModel.Primitives;
+using System.Text.Json;
+
+namespace System.ClientModel.Tests.Client
```

Comment:
Please move these out of the System.ClientModel namespace, as they are not part of this library.

Reply:
They are internal test helpers needed to validate our ship state.